eclipse/src/saros/ui/widgets/viewer/project/ResourceSelectionComposite.java

```diff
@@ -212,20 +212,22 @@ public void run() {
                       if (savedSelectionPresetsCombo.getItemCount() == 0) {
                         SarosView.showNotification(
                             "No stored selections yet",
-                            "You do not have any stored selections yet. Make a selection in the tree above, enter a name here, and press 'Save' to save a selection for easy reusing",
+                            "You do not have any stored selections yet.\n"
+                                + "Make a selection in the tree above, enter a name here.\n"
+                                + "Then press 'Save' to save a selection for easy reusing.",
                             savedSelectionPresetsCombo);
                       } else {
                         SarosView.showNotification(
                             "Error while restoring a selection",
-                            "Could not restore a selection because there is none with the name you entered",
+                            "Could not restore a selection because there is none with the name you entered.",
                             savedSelectionPresetsCombo);
                       }
 
                     } else {
                       SarosView.showNotification(
                           "Selection was restored",
-                          "Your saved selection was successfully restored",
-                          checkboxTreeViewer.getControl());
+                          "Your saved selection was successfully restored.",
+                          savedSelectionPresetsCombo);
```

Comment:
Was the previous usage of `checkboxTreeViewer.getControl()` a wanted difference in behavior or just an error?

Reply:
Error, the message popped up on top left corner of the dialog. instead of the label.